docs/feature-gates.md

```diff
@@ -341,3 +342,14 @@ More documentation will be coming in the future.
 
 User should provide a file server with this feature, and store its authentication credential in a Secret. Antrea
 Controller is required to be configured with the permission to read the Secret.
+
+### Multicluster
+
+`Multicluster` feature enables Antrea multi-cluster Gateway to support multi-cluster Service access across
```

Comment:
Gateway only requires agent side feature gate? Controller side is for cross-cluster NetworkPolicy? Probably talk about them separately.

Reply:
Antrea Multi-cluster Gateway

Reply:
I assume Gateway is not just for Service traffic but can also route Pod - Pod traffic? Maybe: 

enables Antrea Multi-cluster Gateways which route Multi-cluster Service and Pod traffic through tunnels across clusters.

Reply:
Done